appveyor.yml

```diff
@@ -58,6 +68,53 @@ on_failure:
 artifacts:
   - path: dist\*
 
+# scripts to run before deployment
+before_deploy:
+  # install gravitybee/generate the standalone executable
+  - mkdir C:\venv
+  - C:\Python36-x64\Scripts\virtualenv C:\venv
+  - C:\venv\Scripts\activate
+  - python --version
+  - pip install gravitybee satsuki
```

Comment:
i think we should version-pin these build/deploy dependencies in a requirements/deploy.txt. i don't like being surprised when ci/cd dependencies change and break our builds... the install then would be `pip install -r requirements/deploy.txt`.